pages.de/common/head.md

```diff
@@ -0,0 +1,20 @@
+# head
+
+> Gibt den ersten Teil einer Datei aus.
+> Weitere Informationen: <https://www.gnu.org/software/coreutils/head>.
+
+- Gebe die ersten 'count_of_lines' Zeilen der Datei 'filename' aus:
+
+`head -n {{count_of_lines}} {{filename}}`
```

Comment:
Please translate the placeholders, like `count_of_lines` or `filename`.

Reply:
Sure!